app/views/visitor_mailer/booked.html.erb

```diff
@@ -1,22 +1,22 @@
-<p><%= t(".salutation", name: (@visit.visitor_full_name)) %></p>
+<p><%= t(".salutation", name: (@booking_response.visitor_full_name)) %></p>
 
-<p><%= t(".visit_confirmed", prison: (@visit.prison_name)) %></p>
+<p><%= t(".visit_confirmed", prison: (@booking_response.prison_name)) %></p>
 
-<h3><%= format_slot_for_public(@visit.slot_granted) %></h3>
+<h3><%= format_slot_for_public(@booking_response.slot_granted) %></h3>
 
 <p><%= t(".please_arrive") %></p>
 
 <p>
   <%= t('.prisoner') %>
-  <strong><%= @visit.prisoner_anonymized_name %></strong>
+  <strong><%= @booking_response.prisoner_anonymized_name %></strong>
 </p>
 
 <p>
   <%= t('.number') %>
-  <strong><%= @visit.prisoner_number %></strong>
+  <strong><%= @booking_response.prisoner_number %></strong>
 </p>
 
-<% @visit.visitors.allowed.each_with_index do |visitor, index| %>
+<% @booking_response.visitors.allowed.each_with_index do |visitor, index| %>
```

Comment:
This will be wrong if some visitors are rejected as it will show all of the visitors. We need a method on `BookingResponse` that returns only the allowed visitors